libraries/apollo-gradle-plugin/testProjects/convertSchema/schemas/schema-legacy.json

```diff
@@ -0,0 +1,778 @@
+{
+  "__schema": {
```

Comment:
Can we use something that's more precise than legacy to name this schema? For an example using a version from the [spec website](https://spec.graphql.org/): `schema-april-2016-spec.json` ?